public/assets/cbpAnimatedHeader-587a0c0f118d8fe8ee08a9cf34cabdcc68d41775d54d24c4435a5ffca8fbb497.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * cbpAnimatedHeader.js v1.0.0
+ * http://www.codrops.com
+ *
+ * Licensed under the MIT license.
+ * http://www.opensource.org/licenses/mit-license.php
+ *
+ * Copyright 2013, Codrops
+ * http://www.codrops.com
+ */
+$(document).ready(function(){(function(){function n(){window.addEventListener("scroll",function(n){c||(c=!0,setTimeout(e,250))},!1)}function e(){var n=o();n>=i?classie.add(r,"navbar-shrink"):classie.remove(r,"navbar-shrink"),c=!1}function o(){return window.pageYOffset||t.scrollTop}var t=document.documentElement,r=document.querySelector(".navbar-fixed-top"),c=!1,i=300;n()})()});
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'classie' is not defined.<br>'n' is defined but never used.